src/main/java/view_post/interface_adapters/ViewPostViewModel.java

```diff
@@ -1,4 +1,21 @@
 package view_post.interface_adapters;
 
+import view_post.ui.ViewPostView;
+
+import java.beans.PropertyChangeSupport;
+
 public class ViewPostViewModel {
+    private final PropertyChangeSupport support;
+    private final ViewPostView view;
+
+    public ViewPostViewModel(ViewPostView view){
+        this.support = new PropertyChangeSupport(this);
+        this.view = view;
+    }
+
+    public void updateView(ViewPostOutputData data){
+        view.setData(data);
```

Comment:
We don't need this function, as far as I know. It is questionable wrt CA, but we can actually get the data from the property change event